sdk/src/vm.ts

```diff
@@ -153,6 +161,9 @@ export class VM {
      * @experimental
      */
     setUrl: (path: string = '/'): Promise<null> => {
+      if (typeof path !== 'string' || !path.startsWith('/')) {
+        throw new Error(`Invalid argument: expected a path starting with '/', got '${path}'`);
+      }
```

Comment:
Having this method called `setUrl` when you can only pass a path and not a full URL is a bit misleading. The argument being called `path` was not enough to avoid one beta tester from trying to pass a full URL.

There are two possible fixes:

1. Change the method name to `setPath` or `setUrlPath`. (But we kinda like having matching `getUrl` and `setUrl` methods. I'd be open to `setUrlPath`, if we think that's a better choice.)
2. Or keep the method name as `setUrl`, and validate the input.

I went with the second option right now, but we can reconsider.